src/librustc_passes/rvalue_promotion.rs

```diff
@@ -159,6 +179,8 @@ impl<'a, 'gcx> CheckCrateVisitor<'a, 'gcx> {
     /// and transfer it over to the value being matched. This will
     /// then prevent said value from being promoted.
     fn remove_mut_rvalue_borrow(&mut self, pat: &hir::Pat) -> bool {
+        debug!("In remove_mut_rvalue_borrow(), pat: {:#?}", pat);
```

Comment:
Same here and everywhere in the file, I guess. (it's always a good idea to copy the style from somewhere else in the compiler)